profile/README.md

```diff
@@ -47,13 +47,22 @@ We have a few actively developing public reference products:
 - Disabling/enabling L3 offload via sysfs file
 - PCIe SR-IOV support
 
+[meta-xt-prod-devel-rpi5](https://github.com/xen-troops/meta-xt-prod-devel-rpi5) for Raspbery Pi 5
+- Zephyr operated control domain
+- Support dom0less functionality
+- Linux operated driver domain
+- PV hardware backends
+- Linux based guest domain
+- Unikraft based guest domain
+- Virtualized hardware in guest domains: network, USB, SDHD
```

Comment:
It's not clear to me. According to the points above - under guest domains you mean DomUs. For these domains we have the following configuration:
1) we pass gpio to Zephyr guest to control led for a blinky sample
2) we use pvblk and pvnet in Linux based guest domain.
3) SDHD is in Zephyr operated control domain
4) USB is in Linux operated driver domain. 
I think this point should be either dropped completely or rephrased.

Reply:
I have removed this sentence. It was my incorrect interpretation of the product's readme.